src/Temporalio/Client/ListWorkflowsPaginatedOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// Options for <see cref="ITemporalClient.ListWorkflowsPaginatedAsync"/>.
+    /// </summary>
+    /// <param name="PageSize">Number of results per page. Zero means server default.</param>
+    /// <param name="Rpc">RPC options for listing workflows.</param>
+    public record ListWorkflowsPaginatedOptions(int PageSize = 0, RpcOptions? Rpc = null);
```

Comment:
All options classes need to not be records, have zero-arg constructor, and be cloneable. This helps with consistency and fits into some .NET options patterns that can support external configuration.